docs/consume-packages/Package-Restore.md

```diff
@@ -1,242 +1,222 @@
 
-## Package Restore overview
+## Package Restore behavior
 
-Package Restore first installs the direct dependencies of a project as needed, then installs any dependencies of those packages throughout the entire dependency graph.
-
-If a package isn't already installed, NuGet first attempts to retrieve it from the [cache](../consume-packages/managing-the-global-packages-and-cache-folders.md). If the package isn't in the cache, NuGet tries to download the package from all enabled sources in the list at **Tools** > **Options** > **NuGet Package Manager** > **Package Sources** in Visual Studio. During restore, NuGet ignores the order of package sources, and uses the package from whichever source is first to respond to requests. For more information about how NuGet behaves, see [Common NuGet configurations](Configuring-NuGet-Behavior.md). 
-
-> [!Note]
-> NuGet doesn't indicate a failure to restore a package until all the sources have been checked. At that time, NuGet reports a failure for only the last source in the list. The error implies that the package wasn't present on *any* of the other sources, even though errors aren't shown for each of those sources individually.
-
-## Restore packages
+Package Restore tries to install all package dependencies to the state that matches the `<PackageReference>`s in a project file, such as *.csproj*, or `<package>`s in a *packages.config* file. In Visual Studio **Solution Explorer**, you can see the package references under **Dependencies** > **Packages**.
 
-Package Restore tries to install all package dependencies to the correct state matching the package references in your project file (*.csproj*) or your *packages.config* file. (In Visual Studio, the references appear in Solution Explorer under the **Dependencies \ NuGet** or the **References** node.)
-
-1. If the package references in your project file are correct, use your preferred tool to restore packages.
+Package Restore first installs the direct dependencies of a project as needed, then installs any dependencies of those packages throughout the entire dependency graph.
 
-   - [Visual Studio](#restore-using-visual-studio) ([automatic restore](#restore-packages-automatically-using-visual-studio) or [manual restore](#restore-packages-manually-using-visual-studio))
-   - [dotnet CLI](#restore-using-the-dotnet-cli)
-   - [nuget.exe CLI](#restore-using-the-nugetexe-cli)
-   - [MSBuild](#restore-using-msbuild)
-   - [Azure Pipelines](#restore-using-azure-pipelines)
-   - [Azure DevOps Server](#restore-using-azure-devops-server)
+If a needed package isn't already installed, NuGet first attempts to retrieve it from the [NuGet cache](../consume-packages/managing-the-global-packages-and-cache-folders.md). If the package isn't in the cache, NuGet tries to download it from all sources configured in Visual Studio at **Tools** > **Options** > **NuGet Package Manager** > **Package Sources**.
```

Comment:
Maybe we link the how package installation works doc? 

Seems like we've had multiple descriptions of the same concepts and it doesn't shock me they've diverged. 

The tldr is that cache is an oversimplification and a generic buzzword that doesn't really explain much.

Reply:
Regardless of customers calling the global packages folder the "nuget cache", we generally do not. If you run `dotnet nuget locals all -l`, it will list all the special folders NuGet uses, and there is an HTTP cache, but the linked docs filename (managing the global packages and cache folders) calls out "global packages" differently from "cache", so I don't think we should use "NuGet cache" anywhere in our docs.

Reply:
@nkolev92 @zivkan - Although, unfortunately:

![image](https://user-images.githubusercontent.com/20667102/187541155-99bde520-683d-4a84-aec2-5246b2b3fe90.png)

Reply:
Clarified the language.

Reply:
`dotnet nuget locals all -l` outputs:

```text
http-cache: C:\Users\zivkan\AppData\Local\NuGet\v3-cache
global-packages: C:\Users\zivkan\.nuget\packages\
temp: C:\Users\zivkan\AppData\Local\Temp\NuGetScratch
plugins-cache: C:\Users\zivkan\AppData\Local\NuGet\plugins-cache
```

so, there are two cache directories (http and plugins) that will get cleared, and nuget's temp directory, and also the global-packages directory. In other words, the button clears more than just the global packages directory.

Reply:
@zivkan - Right, I had to demonstrate that to myself because the label is misleading. Just saying that if you're trying to get people away from calling it "the cache" or "NuGet Cache," the button label is unfortunate.

Reply:
Honestly, I'm extremely tempted to change the label to "Clear NuGet directories" or "Clear NuGet storage". Changes wouldn't take effect until VS17.4 at the earliest, but I might create that PR today.